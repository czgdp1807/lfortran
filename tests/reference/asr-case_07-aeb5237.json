{
    "basename": "asr-case_07-aeb5237",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/case_07.f90",
    "infile_hash": "993297ce63a56fc4449c9d036888be6e33cde9d51b984fd00f21ef3f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-case_07-aeb5237.stdout",
    "stdout_hash": "7a35b6cfe0aa11a48b87c44a4f08f5fb10295b9b6dafc088742de215",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}