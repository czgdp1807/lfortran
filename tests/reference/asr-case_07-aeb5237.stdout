(TranslationUnit 
    (SymbolTable
        1
        {
            case_07:
                (Program 
                    (SymbolTable
                        2
                        {
                            c:
                                (Variable 
                                    2 
                                    c 
                                    [] 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Character 1 1 () []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                )
                            
                        }) 
                    case_07 
                    [] 
                    [(Select 
                        (Var 2 c) 
                        [(CaseStmt_Range 
                            (StringConstant 
                                "a" 
                                (Character 1 1 () [])
                            ) 
                            (StringConstant 
                                "j" 
                                (Character 1 1 () [])
                            ) 
                            [(Print 
                                () 
                                [(StringConstant 
                                    "one of the first ten letters" 
                                    (Character 1 28 () [])
                                )] 
                                () 
                                ()
                            )]
                        )
                        (CaseStmt_Range 
                            (StringConstant 
                                "l" 
                                (Character 1 1 () [])
                            ) 
                            (StringConstant 
                                "p" 
                                (Character 1 1 () [])
                            ) 
                            [(Print 
                                () 
                                [(StringConstant 
                                    "one of l, m, n, o, p" 
                                    (Character 1 20 () [])
                                )] 
                                () 
                                ()
                            )]
                        )
                        (CaseStmt 
                            [(StringConstant 
                                "z" 
                                (Character 1 1 () [])
                            )] 
                            [(Print 
                                () 
                                [(StringConstant 
                                    "one of z" 
                                    (Character 1 8 () [])
                                )] 
                                () 
                                ()
                            )]
                        )] 
                        [(Print 
                            () 
                            [(StringConstant 
                                "other characters, which may not be letters" 
                                (Character 1 42 () [])
                            )] 
                            () 
                            ()
                        )]
                    )]
                )
            
        }) 
    []
)
